neo/Wallets/KeyPair.cs

```diff
@@ -13,6 +13,8 @@ public class KeyPair : IEquatable<KeyPair>
 
         public UInt160 PublicKeyHash => PublicKey.EncodePoint(true).ToScriptHash();
 
+        public UInt160 ScriptHash => Contract.CreateSignatureRedeemScript(PublicKey).ToScriptHash();
```

Comment:
I think we can get `KeyPair` from wallet by giving the `ScriptHash`. But we can't get `ScriptHash` from `KeyPair`. Because different contracts can have the same `KeyPair`.